### PR TITLE
add tests module snippet

### PR DESCRIPTION
**Pull request by bnjjj**
Request from a friend coming from intellij Rust

### CODE REVIEW

**Comment by matklad** [comment]
yesss, I've been waiting for this for a long time!

bors d+

File: crates/ra_ide/src/completion/complete_snippet.rs [hunk 1]
```diff
@@ -33,6 +33,24 @@ pub(super) fn complete_item_snippet(acc: &mut Completions, ctx: &CompletionConte
...
+        cap,
+        "Test module",
+        "\
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn ${1:test_name}() {
+        $0
```
Comment (bjorn3) [inline crates/ra_ide/src/completion/complete_snippet.rs:47]: Should $0 and $1 be swapped.

Comment (bnjjj) [reply crates/ra_ide/src/completion/complete_snippet.rs:47]: No in fact $1 will be the first cursor position after confirm snippet

**Pull request merged**